src/controllers/graphql/resolvers/user/userResolver.js

```diff
@@ -22,7 +22,12 @@ module.exports = {
         : await authenticatedUserInstance.unFollowSteemUser(args.steemUsernameToUnfollow);
     },
     getProfileInformation: async (_, args) => {
-      return await UserService.getUserInfo(args.name);
+      return await UserService.getUserProfileInfo(args.name);
```

Comment:
Think I've mentioned this before but it's best practice to fetch with `id` in virtually everything (HTTP/REST/GraphQL/SQL etc.) This is working so let it be but we should fetch by `id` whenever available.